http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorSupervisor.scala

```diff
@@ -115,15 +105,14 @@ class ClusterMonitorSupervisor(
   jupyterProxyDAO: JupyterDAO,
   rstudioProxyDAO: RStudioDAO,
   welderProxyDAO: WelderDAO[IO],
-  leonardoService: LeonardoService,
   clusterHelper: ClusterHelper
 )(implicit metrics: NewRelicMetrics[IO], clusterToolToToolDao: ClusterTool => ToolDAO[ClusterTool])
     extends Actor
     with Timers
     with LazyLogging {
   import context.dispatcher
 
-  var monitoredClusters: Set[Cluster] = Set.empty
+  var monitoredClusterIds: Set[Long] = Set.empty
```

Comment:
I know this PR didn't introduce it but not having any implicit or explicit protection on this `var` against race conditions is a bit concerning, no? Doing something like in `ClusterDnsCache.hostToIpMapping` may be an easy remedy.

```scala
private val monitoredClusterIds: ValueBox[Set[Long]] = ValueBox(Set.empty)
```

Reply:
I think this is safe -- this is an actor, and changes to `monitoredClusterIds` is only done in the actor context (not by a future). Actors provide a safe encapsulation of state, so mutable `var`s are allowed. I agree it's probably easy to access this `var` in an unsafe ways, which is one of the things that bothers me about akka..